packages/backend/src/payment-method/ilp/connector/core/middleware/account.ts

```diff
@@ -56,7 +58,15 @@ export function createAccountMiddleware(serverAddress: string): ILPMiddleware {
               IncomingPaymentState.Expired
             ].includes(incomingPayment.state)
           ) {
-            throw new Errors.UnreachableError('destination account is disabled')
+            const errorMessage = 'destination account is disabled'
```

Comment:
maybe "destination account is in an incorrect state" makes more sense?

Reply:
ya thats better, updated